src/Logging.hs

```diff
@@ -15,29 +15,48 @@ module Logging
   , runLoggerLoggingT
 
   -- * Re-exports from "Control.Monad.Logger.Aeson"
+  -- ** Messages
   , Message(..)
   , (.=)
   , Series
+
+  -- ** Thread Context
+  , MonadMask
   , withThreadContext
+  , myThreadContext
+  , Pair
+
+  -- ** Transformer
   , MonadLogger(..)
   , MonadLoggerIO(..)
   , LoggingT
 
   -- ** Common logging functions
   -- | Import "Control.Monad.Logger.Aeson" if you want more
+
+  -- *** Implicit call stack, no 'LogSource'
   , logDebug
   , logInfo
   , logWarn
   , logError
   , logOther
+
+  -- *** Implicit call stack, with 'LogSource'
+  , logDebugNS
```

Comment:
Ever worked in ObjectiveC? This is bringing back some [odd memories](https://www.quora.com/What-is-the-meaning-of-NS-in-objective-C?share=1)...